modules/coins.py

```diff
@@ -260,7 +261,7 @@ def flip(self, event):
             raise utils.EventError("%s: You don't have enough coins to bet" % event["user"].nickname)
 
         chosen_side = random.SystemRandom().choice(list(SIDES.keys()))
-        win = side_name == chosen_side
+        win = side_name == chosen_side or side_name == "heils" and event["user"].nickname == "cygnus"
```

Comment:
ummmm, a bit problematic

Reply:
fist my ass hows that for problematic

Reply:
no I get it now, thanks for the explanation I'll close the comment